mlir/utils/jenkins/Jenkinsfile

```diff
@@ -932,7 +932,13 @@ pipeline {
                         steps {
                             sh 'rm -rf MIGraphX'
                             dir('MIGraphX') {
-                                getAndBuildMIGraphX("-DCMAKE_PREFIX_PATH='${WORKSPACE}/MIGraphXDeps;/MIGraphXDeps;/opt/rocm' -DMIGRAPHX_ENABLE_MLIR=On -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++ -DMIGRAPHX_USE_HIPRTC=Off")
+                                getAndBuildMIGraphX('''
+                                                    -DCMAKE_PREFIX_PATH='${WORKSPACE}/MIGraphXDeps;/MIGraphXDeps;/opt/rocm'
+                                                    -DMIGRAPHX_ENABLE_MLIR=On
+                                                    -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
+                                                    -DMIGRAPHX_USE_HIPRTC=Off
```

Comment:
Would it be possible to make this `-DGPU_TARGETS=${whatever-the-current-arch-is}`?

Reply:
I think it can, I will try to add that the build MIGraphX only for the current GPU architecture.